net/test/test-user.sh.in

```diff
@@ -29,6 +29,16 @@ if [ $1 != "console" -a $1 != "server" -a $1 != "client" ]; then
 	exit
 fi
 
+echo "--------------------Test-user-script------------------------"
+echo -n "Enter (1)Lnet (2)libFab :"
```

Comment:
can we add socket transport ?

Reply:
this is based on endpoints changes. For Sock need to know the endpoints required  and I think we can run the same option of libfab, and the actual transport choice is inside m0_net.